tests/data/aea-config.example.yaml

```diff
@@ -1,4 +1,4 @@
-aea_version: 0.1.1
+aea_version: ">=0.1.1"
```

Comment:
this is to test that we also support version specifiers in `aea-config.yaml` files (and not only mere versions).